### PR TITLE
Feat/preprocess cnn data

### PR DESCRIPTION
**Pull request by SpicyNova**
TLDR; Pre-processes the raw data (original JSON objects) into training and testing data sets for the CNN. 

Both files in the "raw-data-processing" folder are for reference **only**. They were used to make the three CSV files in the "raw-data" folder and construct the test/train data sets. We may need to refer to them in the future so they are there for now and will get deleted eventually.

Main.py has some preliminary stuff for loading the train data set to be fed into the CNN which will be created in a later branch. It loads the card imgs using their URLs and resizes the imgs to be of the same (MxN) size since they will act as inputs for the CNN (inputs to the CNN cannot vary - it accepts a set size). 

blah blah blah...funky data stuff

### CODE REVIEW

**Review by xalluna — APPROVED** [review]

File: data-science/main.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+# LOAD IN TRAINING DATA
+train_data = pd.read_csv('C:/Users/abbyo/Documents/PROJECTS/TcgPocket/data-science/data/train/data.txt')
+print(train_data)
+
+# GET IMGS
+data = []
+for indx, item in train_data.iterrows():
+    print(item.card)
+    resp = requests.get(item.card, headers = {'X-Api-Key': '7ccb4c32-6299-4533-bf47-36f4d2a95117', 'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36'})
```
Comment (zoe-klein) [inline data-science/main.py:19]: definitely need to hide the api key

**Review by DeannaShort — APPROVED** [review]

**Review by zoe-klein — APPROVED** [review]
